tools/isoplot/isoplot.xml

```diff
@@ -0,0 +1,134 @@
+<tool id='isoplot' name='Isoplot: Generate plots from isocor output' version='1.3.0+galaxy0' profile='20.01'>
+    <requirements>
+        <requirement type='package' version='1.3.0'>isoplot</requirement>
+    </requirements>
+    <command><![CDATA[
+         isoplot '$datafile' 'galaxy' '$format' --value '$value' --template '$template_file' $SP.barplot $SP.meaned_barplot 
+         $SP.static_heatmap $SP.static_clustermap $IP.interactive_barplot $IP.interactive_meanplot $IP.interactive_heatmap
+#if $unstack:
+    -s
+#end if
+#if $verbosity:
+    -v
+#end if
+#if $annotation:
+    -a
+#end if
+-g -z '$plots_output' > $data_output 2> '$log_output'
+    ]]></command>
+    <inputs>
+        <param name='datafile' type='data' format='csv, tabular' multiple='true' label='Upload data file' />

```

Comment:
Can you find a "better" label?
Or an help on the potentiel origin of this file?

Reply:
I have changed the input data labels to be more comprehensive. For the collections though, I feel that a codebase change is needed, so if possible I would like to wait for the next release before implementing them. Is this possible?